test/e2e/mtls/standalone_universal.go

```diff
@@ -0,0 +1,139 @@
+package mtls
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kumahq/kuma/pkg/config/core"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func StandaloneUniversal() {
+	var universal Cluster
+	var universalOpts = KumaUniversalDeployOpts
+
+	BeforeEach(func() {
+		clusters, err := NewUniversalClusters([]string{Kuma1}, Silent)
+		Expect(err).ToNot(HaveOccurred())
+
+		universal = clusters.GetCluster(Kuma1)
+		Expect(Kuma(core.Standalone, universalOpts...)(universal)).To(Succeed())
+		Expect(universal.VerifyKuma()).To(Succeed())
+	})
+
+	E2EAfterEach(func() {
+		Expect(universal.DeleteKuma(universalOpts...)).To(Succeed())
+		Expect(universal.DismissCluster()).To(Succeed())
+	})
+
+	createMesh := func(name, mode string) {
+		meshYaml := fmt.Sprintf(
+			`
+type: Mesh
+name: %s
+mtls:
+  enabledBackend: ca-1
+  backends:
+  - name: ca-1
+    type: builtin
+    mode: %s`, name, mode)
+		Expect(YamlUniversal(meshYaml)(universal)).To(Succeed())
+	}
+
+	runDemoClient := func(mesh string) {
+		demoClientToken, err := universal.GetKuma().GenerateDpToken(mesh, "demo-client")
+		Expect(err).ToNot(HaveOccurred())
+		Expect(
+			DemoClientUniversal(AppModeDemoClient, mesh, demoClientToken, WithTransparentProxy(true))(universal),
+		).To(Succeed())
+	}
+
+	runTestServer := func(mesh string) {
+		echoServerToken, err := universal.GetKuma().GenerateDpToken(mesh, "test-server")
+		Expect(err).ToNot(HaveOccurred())
+
+		Expect(TestServerUniversal("test-server", mesh, echoServerToken,
+			WithArgs([]string{"echo", "--instance", "universal-1"}))(universal)).To(Succeed())
+	}
+
+	checkInsideMeshConnection := func() {
+		Eventually(func() bool {
+			stdout, _, err := universal.Exec("", "", "demo-client", "curl", "-v", "-m", "8", "--fail", "test-server.mesh")
+			if err != nil {
+				return false
+			}
+			return strings.Contains(stdout, "HTTP/1.1 200 OK")
+		}, "30s", "1s").Should(BeTrue())
```

Comment:
there is no point in checking 200 OK. If you have `--fail`, it will fail for other codes, so how about just
```
Eventually(func() error {
    _, _, err := universal.Exec("", "", "demo-client", "curl", "-v", "-m", "8", "--fail", "test-server.mesh")
    return err
}, "30s", "1s").ShouldNot(HaveOccured())
```

also, why `-m 8`?